addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-group-type id="arrivalRange">
+        <label>Range</label>
+        <description>Range for a route event</description>
+        <channels>
+            <channel id="arrival" typeId="arrival" />
+            <channel id="departure" typeId="departure" />
+            <channel id="event" typeId="routeEvent" />
+            <channel id="update" typeId="update" />
+        </channels>
+    </channel-group-type>
+
+    <channel-type id="arrival">
+        <item-type>DateTime</item-type>
+        <label>Arrival Time</label>
+        <description>The arrival time of the event</description>
```

Comment:
It is the arrival time of the bus/subway/railway (and you can configure an offset, yes). So if "bus" is the wrong word, simply write "The arrival time" here.